js/main.js

```diff
@@ -1,13 +1,53 @@
+const formData = localStorage.getItem('form-data') ? JSON.parse(localStorage.getItem('form-data')) : {
+  Full_name: '',
+  email: '',
+  message: '',
+};
+
+let form;
+
+function loadData(tempFormData) {
+  Object.keys(tempFormData).forEach((inputName) => {
+    form[inputName].value = tempFormData[inputName];
+  });
+}
+
+function saveData(tempFormData) {
+  localStorage.setItem('form-data', JSON.stringify(tempFormData));
+}
+
 function menuToggle() {
   document.querySelector('#nav-menu').classList.toggle('toggle-mobile-menu');
 }
 
 window.addEventListener('load', () => {
+  form = document.getElementById('contact-me-form');
+  loadData(formData);
   document.querySelector('#menu-btn').addEventListener('click', () => {
     menuToggle();
   });
 
   document.querySelector('#nav-menu').addEventListener('click', () => {
     menuToggle();
   });
+
+  form.addEventListener('submit', (event) => {
+    if (document.querySelector('small')) form.removeChild(document.querySelector('small'));
+    if (form.email.value !== form.email.value.toLowerCase()) {
+      event.preventDefault();
+      const listItem = document.createElement('li');
+      const errorMessage = document.createElement('small');
+      errorMessage.innerText = 'Email Contains Capital Letters';
+      listItem.append(errorMessage);
+      form.querySelector('ul').append(listItem);
+    }
+  });
```

Comment:
## [OPTIONAL]
- [ ] Kindly I would suggest you use to fix this issue on your validation. Currently when I double click on the submit button this is being submitted although your email input contains an uppercase letter.